package.json

```diff
@@ -64,6 +64,7 @@
     "chai-as-promised": "^7.1.1",
     "clipboard-polyfill": "^3.0.3",
     "dirty-chai": "^2.0.1",
+    "dom-accessibility-api": "^0.5.14",
```

Comment:
This is the dependency that `@testing-library` uses under the hood for its accessibility querying, but (AFAICT) not everything is exposed (particularly, finding description related to a specific element, as in the spec above). So not really a _new_ dependency, but now directly referenced.